### PR TITLE
Refactor something about SMS & Sentinel

### PR DESCRIPTION
**Pull request by fangjian0423**
### Describe what this PR does / why we need it

as titled.

### CODE REVIEW

File: spring-cloud-alibaba-docs/src/main/asciidoc-zh/sms.adoc [hunk 1]
```diff
@@ -10,17 +10,10 @@ Spring Cloud Alibaba 已经发布了 0.2.2.BUILD-SNAPSHOT 版本，需要首先
...
-            <artifactId>spring-cloud-dependencies</artifactId>
-            <version>Finchley.SR2</version>
-            <type>pom</type>
-            <scope>import</scope>
-        </dependency>
         <dependency>
             <groupId>org.springframework.cloud</groupId>
             <artifactId>spring-cloud-alibaba-dependencies</artifactId>
-            <version>0.2.2.BUILD-SNAPSHOT</version>
+            <version>0.2.0.RELEASE</version>
```
Comment (flystar32) [inline spring-cloud-alibaba-docs/src/main/asciidoc-zh/sms.adoc]: doc 还是保持一致吧， 发布的时候会进行全文替换的

File: spring-cloud-alicloud-sms/src/main/java/org/springframework/cloud/alicloud/sms/base/DefaultAlicomMessagePuller.java [hunk 2]
```diff
@@ -33,12 +25,21 @@
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.ScheduledThreadPoolExecutor;
 
+import org.apache.commons.lang3.concurrent.BasicThreadFactory;
+import org.apache.commons.logging.Log;
```
Comment (flystar32) [inline spring-cloud-alicloud-sms/src/main/java/org/springframework/cloud/alicloud/sms/base/DefaultAlicomMessagePuller.java]: slf4j ？

File: spring-cloud-alibaba-docs/src/main/asciidoc-zh/sms.adoc (hunk 1 above)
Comment (fangjian0423) [reply spring-cloud-alibaba-docs/src/main/asciidoc-zh/sms.adoc]: Yes.

But not only here, but other places have to change. Will create another pr to update docs.

File: spring-cloud-alicloud-sms/src/main/java/org/springframework/cloud/alicloud/sms/base/DefaultAlicomMessagePuller.java (hunk 2 above)
Comment (fangjian0423) [reply spring-cloud-alicloud-sms/src/main/java/org/springframework/cloud/alicloud/sms/base/DefaultAlicomMessagePuller.java]: I think this is the raw code in sms sdk, we just copy it and do not modify it.

File: spring-cloud-alibaba-docs/src/main/asciidoc-zh/sms.adoc (hunk 1 above)
Comment (xiaolongzuo) [inline spring-cloud-alibaba-docs/src/main/asciidoc-zh/sms.adoc]: 这个地方为什么改成了0.2.0

File: spring-cloud-alicloud-sms/src/main/java/org/springframework/cloud/alicloud/sms/base/TokenGetterForAlicom.java [hunk 3]
```diff
@@ -124,8 +125,8 @@ public TokenForAlicom getTokenByMessageType(String messageType, String queueName
 					CloudAccount account = new CloudAccount(token.getTempAccessKeyId(),
 							token.getTempAccessKeySecret(), mnsAccountEndpoint,
 							token.getToken());
-					// logger.warn("ak:"+token.getTempAccessKey());
-					// logger.warn("token:"+token.getToken());
+					// log.warn("ak:"+token.getTempAccessKey());
```
Comment (xiaolongzuo) [inline spring-cloud-alicloud-sms/src/main/java/org/springframework/cloud/alicloud/sms/base/TokenGetterForAlicom.java]: 注释掉的代码最好直接删掉。

**Pull request merged**